### ISSUE TITLE
OAuth access token

### ISSUE THREAD

**Issue by simov** [seladb/StarTrack-js#3]
Just FYI you can use the showcase app of one of my projects - https://grant.outofindex.com/github to get access token for GitHub. It's much safer to use access token then your actual password.

The login process can also be automated to login through my showcase app when the user clicks a button in your own app, and be redirected back to it. The access token will be returned in the querystring from where you can extract it and put it localStorage. But that would require a bit more work I guess.

Anyway, I hope that helps, feel free to close this issue.

**Comment by seladb** [comment]
Thanks for providing these details. StarTrack-js actually offers both options: either provide username + password or provide GitHub access token, so the user can choose between the two. Please take a look at this screenshot:

![image](https://user-images.githubusercontent.com/9059541/50274123-48822080-03f1-11e9-9102-8fc207d9d611.png)

**Comment by simov** [comment]
Yes, I know. My thinking was that probably a lot of people won't have an access token, and will provide their password instead, which is the worse case. So I wanted to point out an easy way to get an access token.

**Comment by seladb** [comment]
Got it. That's a good idea however your showcase requires logging in to GitHub and giving authorization for your user to access the StarTrack's users GitHub account, which is also not that safe.

The unique thing about StarTrack-js is that everything runs on the client side so users' credentials aren't sent anywhere (other than GitHub itself for fetching the data)
